### ISSUE TITLE
Inconsistent prettier configuration

### ISSUE THREAD

**Issue by darekkay** [mwakerman/kanbanist#13]
In your README, you're using the following prettier configuration:

```
prettier --print-width 120 ---tab-width 4 -single-quote --jsx-bracket-same-line --arrow-parens avoid --trailing-comma es5 --write src/**/*.js
```

However, all JavaScript files in this repository use 2 spaces for indent.

I would like to open a PR with a `.prettierrc` file and a `format` script to enforce your coding styleguides, but this would reformat every single file.

So my question is - would you rather change the code styling to 2 spaces or reformat all files with 4 spaces?

**Comment by mwakerman** [comment]
You’re correct! I was very inconsistent with applying prettier and had it working as a git commit hook on modified files for a time but then it broke and I never fixed it again. I’d welcome a PR that standardises the styling. My preference is 4 spaces for JavaScript and a big style-only commit is fine :)
